pages/home.html

```diff
@@ -21,20 +27,18 @@
     Bite-sized
   </h3>
   <p>
-    Explorations are easy to create and quick to play. They can be published in
-    the exploration gallery, shared on social media, or embedded on your
-    webpage.
+    Easy to create and quick to play, explorations can also be published in the exploration gallery, shared on social media, or embedded on your webpage.
```

Comment:
Interesting! In that case, I think it makes sense to go with the original one, since a more diverse group of people prefer it. Would you agree?